src/pages/ProfileLayout/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { useTranslation } from 'react-i18next';
+import { Outlet, useLocation } from 'react-router-dom';
+
+import { Avatar, Grid } from '@mui/material';
+import { Box } from '@mui/system';
+
+import SectionTitle from 'src/components/shared/SectionTitle';
+import { useAppSelector } from 'src/redux/hooks';
+import { selectUserAvatar, selectUserName } from 'src/redux/user/userSlice';
+
+import PhotoUploadForm from '../ProfilePage/PhotoUploadForm';
+
+import ProfileMenu from './ProfileMenu';
+import {
+  AvatarWrapper,
+  NameTitle,
+  OutletWrapper,
+  SideBarWrapper,
+  SubTitle,
+} from './styles';
+
+function ProfileLayout() {
+  const { t } = useTranslation();
+  const location = useLocation();
+  const userName = useAppSelector(selectUserName) ?? t('profile.name');
+  const userAvatar = useAppSelector(selectUserAvatar) ?? '';
+
+  const isProfileDetails = location.pathname === '/profile/details';
```

Comment:
I saw that you have pathes in constant, the I assume this one also should be there

Reply:
fixed